src/pl/ogladajanime/src/eu/kanade/tachiyomi/animeextension/pl/ogladajanime/OgladajAnime.kt

```diff
@@ -0,0 +1,319 @@
+
+    // =========================== Anime Details ============================
+    override fun animeDetailsParse(document: Document): SAnime {
+        return SAnime.create().apply {
+            // status = document.selectFirst("div.toggle-content > ul > li:contains(Status)")?.let { parseStatus(it.text()) } ?: SAnime.UNKNOWN // Nie pamietam kiedyś sie to naprawi.
+            description = document.selectFirst("p#animeDesc")?.text()
+            genre = document.select("div.row > div.col-12 > span.badge[href^=/search/name/]").joinToString(", ") {
+                it.text()
+            }
+            author = document.select("div.row > div.col-12:contains(Studio:) > span.badge[href=#]").joinToString(", ") {
+                it.text()
+            }
+        }
+    }
+
+    // ============================== Episodes ==============================
+
+    override fun episodeListRequest(anime: SAnime): Request {
+        val url = baseUrl + anime.url
+        return GET(url, apiHeaders)
+    }
+
+    override fun episodeListParse(response: Response): List<SEpisode> {
+        val document = response.asJsoup()
+        val episodeList = mutableListOf<SEpisode>()
+
+        document.select("ul#ep_list > li").forEach { element ->
+            val episode = SEpisode.create()
+            val episodeNumber = element.attr("value").toFloatOrNull() ?: 0f
+            val episodeText = element.select("div > div > p").text()
+
+            val episodeImg = element.select("div > img").attr("alt")
+            val check = element.select("div > img")
+
+            if (check.isNotEmpty()) {
+                episode.name = if (episodeText.isNotEmpty()) {
+                    "[${episodeNumber.toInt()}] $episodeText ($episodeImg)"
+                } else {
+                    "Episode ${episodeNumber.toInt()} ($episodeImg)"
+                }
+                episode.episode_number = episodeNumber
+                episode.url = element.attr("ep_id")
+                episodeList.add(episode)
+            }
+        }
+        return episodeList.reversed()
+    }
+
+    override fun episodeListSelector(): String = throw UnsupportedOperationException()
+
+    override fun episodeFromElement(element: Element): SEpisode = throw UnsupportedOperationException()
+
+    // ============================ Video Links =============================
+
+    private fun getPlayerUrl(player: String): String {
+        val body = FormBody.Builder()
+            .add("action", "change_player_url")
+            .add("id", player)
+            .build()
+        return client.newCall(POST("$baseUrl/manager.php", apiHeaders, body))
+            .execute()
+            .use { response ->
+                response.body.string()
+                    .substringAfter("\"data\":\"")
+                    .substringBefore("\",")
+                    .replace("\\", "")
+            }
+    }
+    override suspend fun getVideoList(episode: SEpisode): List<Video> {
```

Comment:
I don't see any reason to override `getVideoList`, consider using `videoListRequest` and `videoListParse` instead